modules/gridBidAdapter.js

```diff
@@ -136,8 +151,19 @@ export const spec = {
       if (impObj.banner || impObj.video) {
         imp.push(impObj);
       }
+
+      if (!forceBidderName && forceBidder && ALIAS_CONFIG[forceBidder]) {
+        forceBidderName = forceBidder;
+      }
     });
 
+    forceBidderName = config.getConfig('forceBidderName') || forceBidderName;
```

Comment:
please follow the conventions here

https://github.com/prebid/Prebid.js/issues/8129

Reply:
Hello @patmmccann ! In this PR TrustX adapter migrate to alias for grid bid adapter and when gird bid adapter catch bidderCode === 'trustx', it sent request with alternative parameters. Can you suggest any solution to  to fix case, when publisher set alias for trustx - pbjs.aliasBidder('trustx', 'mybidder'); and we need to sent request as for trustx? How grid bid adapter can find that 'mybidder' is alias exactly for 'trustx', not for 'grid'?

Reply:
If i right understand this feature https://github.com/prebid/Prebid.js/issues/8129 can't resolve this problem.

Reply:
Tyvm